Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -51,12 +51,22 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 					connectInterfaces(obj, rayOrigin);
 			}
 
-			internal void DisconnectInterfaces(object obj)
+			internal void DisconnectInterfaces(object obj, InputDevice device)
```

Comment:
@AndrewTHEManeri - if you're going to fix, then take a look at my original comment.